### DISCUSSION TITLE
Sort order option for relationship picker

### DISCUSSION THREAD

**Discussion by gerhardmpl** [Budibase/budibase#5574] [Ideas]
Somehow following #4579 I would like to propose a `sort order` option (ascending or descending) in the relationship picker component settings in the design section. It would reduce complexity in design (compared to #4579) and increase usability of the component to make sure that the options are sorted (which they are not at the moment).

**Comment by melohagan** [comment]
This would be a nice feature! I've created a request from this discussion here: https://github.com/Budibase/budibase/issues/5582
